Gaomon_M106K/descriptors.txt

```diff
@@ -0,0 +1,90 @@
+
+Bus 001 Device 006: ID 256c:006e  
+Device Descriptor:
+  bLength                18
+  bDescriptorType         1
+  bcdUSB               1.10
+  bDeviceClass            0 (Defined at Interface level)
+  bDeviceSubClass         0 
+  bDeviceProtocol         0 
+  bMaxPacketSize0        64
+  idVendor           0x256c 
+  idProduct          0x006e 
+  bcdDevice            0.00
+  iManufacturer           5 (error)
+  iProduct                6 (error)
```

Comment:
This is simply using the Huion's generic VID:PID, which they use for all their tablets, and doesn't have any manufacturer or product strings, so we'll just have to name it as "Gaomon M106K" which you did.